geniesp/metafiles.py

```diff
@@ -0,0 +1,262 @@
+                filename=cbio_file,
+            )
+        elif cbio_file.startswith("data_clinical_patient"):
+            create_clinical_meta_file(
+                study_identifier=study_identifier,
+                alteration_type="CLINICAL",
+                datatype="PATIENT_ATTRIBUTE",
+                filename=cbio_file,
+            )
+        elif cbio_file.startswith("data_gene_matrix"):
+            create_clinical_meta_file(
+                study_identifier=study_identifier,
+                alteration_type="GENE_PANEL_MATRIX",
+                datatype="GENE_PANEL_MATRIX",
+                filename=cbio_file,
+            )
+        elif cbio_file.startswith(
+            [
+                "data_timeline_cancer_diagnosis",
+                "data_timeline_imaging",
+                "data_timeline_medonc",
+                "data_timeline_pathology",
+                "data_timeline_sample_acquisition",
+                "data_timeline_sequencing",
+                "data_timeline_treatment",
+            ]
+        ):
+            create_clinical_meta_file(
+                study_identifier=study_identifier,
+                alteration_type="CLINICAL",
+                datatype="TIMELINE",
+                filename=cbio_file,
+            )
+        elif cbio_file.startswith("data_CNA"):
+            create_genomic_meta_file(
+                study_identifier=study_identifier,
+                alteration_type="COPY_NUMBER_ALTERATION",
+                datatype="DISCRETE",
+                stable_id="cna",
+                profile_name="Copy-number alterations",
+                profile_description="Copy-number alterations",
+                filename=cbio_file,
+            )
+        elif cbio_file.startswith("data_fusions"):
+            create_genomic_meta_file(
+                study_identifier=study_identifier,
+                alteration_type="FUSION",
+                datatype="FUSION",
+                stable_id="fusion",
+                profile_name="Fusions",
+                profile_description="Fusions",
+                filename=cbio_file,
+            )
+        elif cbio_file.startswith("data_mutations_extended"):
+            create_genomic_meta_file(
+                study_identifier=study_identifier,
+                alteration_type="MUTATIONS_EXTENDED",
+                datatype="MAF",
+                stable_id="mutations",
+                profile_name="Mutations",
+                profile_description="Mutation data from next-gen sequencing.",
+                filename=cbio_file,
+            )
+        elif cbio_file.endswith("data_cna_hg19.seg"):
+            create_seg_meta_file(
+                study_identifier=study_identifier,
+                reference_genome_id="hg19",
+                description="Segment data for the genie study",
+                filename=cbio_file,
+            )
+        else:
+            print(f"{cbio_file} does not have associated metafile")
+        create_meta_study(
```

Comment:
I think its a bit awkward that this function creates a meta_study file every time.

Reply:
We can also extract the mg_release from the `_MG_RELEASE_SYNID`.